instrument/VULCAN_Definition_2021-01-01.xml

```diff
@@ -0,0 +1,1468 @@
+<?xml version='1.0' encoding='ASCII'?>
+<instrument xmlns="http://www.mantidproject.org/IDF/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 http://schema.mantidproject.org/IDF/1.0/IDFSchema.xsd"
+            name="VULCAN" valid-from   ="2021-01-01 00:00:01"
+                          valid-to     ="2022-05-15 00:00:00"
```

Comment:
Midnight is confusing for a time. I know that international standards are clear, but there is not a ton of upside for making people finding it.
```suggestion
                          valid-to     ="2022-05-14 23:59:59"
```